nl_NL/LC_MESSAGES/youtube.po

```diff
@@ -4,191 +4,191 @@ msgstr ""
 
 msgid "TITLE_DESC_MISSING"
-msgstr "Your Flipnote needs to have a title and description before you can upload it."
+msgstr "Je flipnote heeft eerst een titel en beschrijving nodig voordat je het naar youtube kunt uploaden"
 
 msgid "TITLE_DESC_ADD_FIRST"
-msgstr "Please go back and add these before continuing."
+msgstr "Ga terug en verbeter dit eerst voordat je doorgaat"
 
 # A URL is displayed after this:
 msgid "HOW_TO_LINK_FROM_THEATRE"
-msgstr "To link your YouTube channel, visit this page:"
+msgstr "Om je youtube kanaal te koppen ga naar deze pagina:"
 
 msgid "HOW_TO_REFRESH_FROM_THEATRE"
-msgstr "To update your channel, visit this page:"
+msgstr "Om je kanaal te updaten ga naar deze pagina:"
 
 # Preview upload details before submitting
 msgid "PREVIEW_TITLE"
-msgstr "Title"
+msgstr "Titel"
 
 msgid "PREVIEW_DESC"
-msgstr "Description"
+msgstr "Beschrijving"
 
 msgid "PREVIEW_EXTRA_DESC"
-msgstr "Extra Description"
+msgstr "Extra Beschrijving"
 
 msgid "PREVIEW_EXTRA_DESC_EXPLANATION"
-msgstr "This text will display when viewers click on 'Show more' in your video's description."
+msgstr "Deze tekst wordt weergegeven wanneer kijkers in de beschrijving van je video op 'Meer weergeven' klikken."
 
 msgid "TAGS"
 msgstr "Tags"
 
 msgid "NONE"
-msgstr "None"
+msgstr "Geen"
 
 msgid "YOUTUBE_LOOP_COUNT"
-msgstr "Number of plays"
+msgstr "Hoeveelheid loops"
 
 msgid "YOUTUBE_LOOP_COUNT_EXPLANATION"
-msgstr "Number of times a looped Flipnote will repeat. If 1, the Flipnote will only play once."
+msgstr "Aantal keren dat een Flipnote in een lus wordt herhaald. Indien 1 zal de Flipnote maar één keer worden afgespeeld."
 
 msgid "YOUTUBE_VISIBILITY"
-msgstr "Visibility"
+msgstr "Zichtbaarheid"
 
 msgid "YOUTUBE_VISIBILITY_PUBLIC"
-msgstr "Public"
+msgstr "openbaar"
 
 msgid "YOUTUBE_VISIBILITY_UNLISTED"
-msgstr "Unlisted"
+msgstr "Verborgen"
 
 msgid "YOUTUBE_VISIBILITY_PRIVATE"
-msgstr "Private"
+msgstr "privé"
 
 msgid "SUBMIT"
-msgstr "Submit"
+msgstr "Verstuur"
 
 # Upload status
 msgid "UPLOAD_STATUS"
 msgstr "Upload Status"
 
 msgid "YOUTUBE_PENDING_UPLOAD"
-msgstr "Your Flipnote is currently waiting to be uploaded. Check back in a few minutes."
+msgstr "Je flipnote is op dit moment aan het wachten om geupload te worden. kijk over een paar minuutjes opnieuw"
 
 # Position in queue: 4
 msgid "POSITION_IN_QUEUE"
-msgstr "Position in queue: %d"
+msgstr "Positie in wachtrij: %d"
 
 msgid "THEATRE_STATUS_TRACKER"
-msgstr "Watch your live Upload Tracker here:"
+msgstr "Bekijk je live upload tracker hier"
 
 msgid "REVIEW_DETAILS_BEFORE_UPLOAD"
-msgstr "Please review these options before starting your upload."
+msgstr "Bekijk deze opties voordat je je flipnote gaat uploaden"
 
 # ...do so, visit https://flipnot.es/ABC123/yt
 msgid "CHANGE_OPTIONS_ON_THEATRE"
-msgstr "You can change these options before upload. To do so, visit:"
+msgstr "Je kunt deze opties wijzigen voordat je je flipnote upload om dat te doen ga naar:"
```

Comment:
Why is "Flipnote" mentioned here at all? If it has to be there then the first letter should be capitalized. Also, this seems to be combining two sentences into one again, it should be two.